### PR TITLE
Autogenerate diagram for RequestStatus state machine

### PR DESCRIPTION
**Pull request by bloodearnest**
Integrate with just fix/check like other tools.

### CODE REVIEW

File: justfile [hunk 1]
```diff
@@ -121,7 +122,7 @@ mypy *ARGS: devenv
 
 
 # fix formatting and import sort ordering
```
Comment (inglesp) [inline justfile:124]: This comment's now out of date

**Review by inglesp — COMMENTED** [review]
Great!

How are we going to make sure it doesn't get stale?  Currently it looks like it relies on a developer running `just fix` at the right time, and committing the changes.

**Comment by bloodearnest** [comment]
> Great!
> 
> How are we going to make sure it doesn't get stale? Currently it looks like it relies on a developer running `just fix` at the right time, and committing the changes.

I included it in just check for exactly that reason - it will fail if its out of date. And we run `just check` in CI

File: justfile (hunk 1 above)
Comment (bloodearnest) [reply justfile]: fixed

**Comment by inglesp** [comment]
> I included it in just check for exactly that reason - it will fail if its out of date. And we run `just check` in CI

Ah yes, sorry, missed that

**Review by inglesp — APPROVED** [review]

**Pull request merged**